ui/helpers/utils/build-types.js

```diff
@@ -20,7 +20,7 @@ export function isBeta() {
 // the current metamask version (i.e. main, beta, etc.)
 export function getBuildSpecificAsset(assetName) {
   const buildType = process.env.METAMASK_BUILD_TYPE;
-  if (!assetList[buildType]?.[assetName]) {
+  if (!assetList[buildType] || !(assetName in assetList[buildType])) {
```

Comment:
Nit: There's a [proposed lint change](https://github.com/MetaMask/eslint-config/pull/207) that would prevent using the `in` operator because it looks up the entire prototype chain, which can be unexpected and confusing.

These two alternatives are functionally equivalent and would pass that rule:
```suggestion
  if (!assetList[buildType] || !Object.keys(assetList[buildType]).includes(assetName)) {
```
or
```suggestion
  if (!assetList[buildType] || Reflect.hasOwnProperty.call(assetList[buildType], assetName)) {
```